letsencrypt-auto

```diff
@@ -14,7 +14,12 @@ VENV_PATH=${VENV_PATH:-"$XDG_DATA_HOME/$VENV_NAME"}
 VENV_BIN=${VENV_PATH}/bin
 
 if test "`id -u`" -ne "0" ; then
-  SUDO=sudo
+  if ! type "sudo" > /dev/null; then
+    su
```

Comment:
On second thought, environments that don't support sudo (I guess Android is a good example) wouldn't ask for passwords every single time a command is added, as applications such as SuperSU manage that automatically, and they provide the application with root access whenever requested later, although I'm not really sure who'd possibly use Let's Encrypt on an Android device...

However, prepending the commands with ```su  -c "``` and appending them with an extra ```"``` doesn't seem like a really bad idea. Is there such a system where ```sudo``` is not available, apart from Android, so that one can test something like that?